docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -609,7 +609,7 @@ Builtin variables are available under the `builtin.` prefix. Some examples:
   - all the reference must be in the same namespace of `metadata.Namespace`
   - `spec.workers.machineDeployments[i].class` field must be unique within a ClusterClass.
   - `ClusterClassVariable`:
-    - names must be unique, not empty and not equal to `builtin`
+    - names must be unique, not empty and not start with `builtin`
```

Comment:
@sbueringer when we consider complex types, this should be changed into "should note be `builtin`" (reserved variable name)
Same below

Reply:
Yeah, but I think we can already go back now to `should not be builtin` or `should not equal builtin` given that https://github.com/kubernetes-sigs/cluster-api/pull/5615#discussion_r749990826 will solve the `builtin.<something` variable name problem

Reply:
I think we're now at: "names must be: unique, not empty, not equal to `builtin` and should not contain dots"
(same below for updates)